lib/id3c/db/__init__.py

```diff
@@ -56,37 +56,24 @@ def mint_identifiers(session: DatabaseSession, name: str, n: int) -> Any:
 
     started = datetime.now()
```

Comment:
```suggestion

```
Don't believe this line is needed anymore with stats being calculated in postgres.